rust/private/rust.bzl

```diff
@@ -796,6 +796,15 @@ _rust_binary_attrs = {
         cfg = "exec",
         allow_single_file = True,
     ),
+    "crate_type": attr.string(
+        doc = _tidy("""
+            Crate type that will be passed to `rustc` to be used for building this crate.
+
+            This option is a temporary workaround and should be used only when building
+            for WebAssembly targets (//rust/platform:wasi and //rust/platform:wasm).
+        """),
```

Comment:
Let me know if you want to document this feature or if I should remove the `doc` field.